launching-containers/building/registry-authentication/index.md

```diff
@@ -30,6 +30,9 @@ Here's what an example looks like with credentials for docker's public index and
 }
 ```
 
+Note: The auth token is a base64 encoded string: `base64(<username>:<password>)`.
```

Comment:
I'd like to avoid having to say "note", over time it will make it harder to read as you have more notes. What if you said:

> If you want to populate these values without running `docker login`, the auth token is a base64 encoded string: `base64(<username>:<password>)`.